int32 agent1_value

uint16 agent1_posx
uint16 agent1_posy
uint16 agent1_post

uint16 agent1_ballx
uint16 agent1_bally
uint16 agent1_ball_detected

uint8 agent1_pass
uint8 agent1_condition